data/mods/blazemod/blaze_weapons_parts.json

```diff
@@ -115,6 +120,7 @@
         "broken_color" : "dark_gray",
         "damage_modifier" : 80,
         "durability" : 400,
+        "fuel_type" : "50",
```

Comment:
A note about this: the game takes the ammo type now directly from the gun of the turret, putting it here as fuel type is not needed anymore. It is actually wrong because fuel types are item types, not ammo types (see #12411 for the reaching about this).

And a final note  (you may have already done this), please go into the vehicle info and try to install a part and scroll through *all* listed item. This should not not give any debug messages. Listing an ammo type (which is not by chance also an item type) here will show a debug message.

In short: just remove the fuel_type entry from all parts with the TURRET flag (it is only required for parts with the FUEL_TANK or the ENGINE flag)